doc/rtd/conf.py

```diff
@@ -119,6 +119,15 @@
 autosectionlabel_prefix_document = True
 autosectionlabel_maxdepth = 2
 
+# Sphinx-linkcheck config
+linkcheck_ignore = [   r"http://instance-data.*", r"https://powersj.io.*",
```

Comment:
Great thanks @itsaviral2609, one more ignore to add and then I think we can drop the high-water mark for # of broken links from 6 to 3

```diff
diff --git a/doc/rtd/conf.py b/doc/rtd/conf.py
index 0dc3b4839..03c8688d2 100644
--- a/doc/rtd/conf.py
+++ b/doc/rtd/conf.py
@@ -120,8 +120,9 @@ autosectionlabel_prefix_document = True
 autosectionlabel_maxdepth = 2
 
 # Sphinx-linkcheck config
-linkcheck_ignore = [   r"http://instance-data.*", r"https://powersj.io.*",
-                      r"http://169.254.169.254.*", r"http://10.10.0.1.*"
+linkcheck_ignore = [
+    r"http://\[fd00:ec2::254.*", r"http://instance-data.*",
+    r"https://powersj.io.*", r"http://169.254.169.254.*", r"http://10.10.0.1.*"
 ]
 linkcheck_anchors_ignore_for_url = (
     r"https://github.com/canonical/cloud-init.*",
```